datasetinsights/datasets/synthetic.py

```diff
@@ -27,17 +27,19 @@ def read_bounding_box_3d(annotation, label_mappings=None):
 
     for b in annotation:
         label_id = b["label_id"]
-        translation = b["translation"]
-        translation = [translation["x"], translation["y"], translation["z"]]
-        size = b["size"]
-        size = [size["x"], size["y"], size["z"]]
+        translation = (
+            b["translation"]["x"],
+            b["translation"]["y"],
+            b["translation"]["z"],
+        )
+        size = (b["size"]["x"], b["size"]["y"], b["size"]["z"])
         rotation = b["rotation"]
         rotation = Quaternion(
-            b=rotation["x"], c=rotation["y"], d=rotation["z"], a=rotation["w"]
+            x=rotation["x"], y=rotation["y"], z=rotation["z"], w=rotation["w"]
         )
 
-        if label_mappings and label_id not in label_mappings:
-            continue
+        # if label_mappings and label_id not in label_mappings:
```

Comment:
If it's removed, it will get an error when there is something wrong with the label id. Or you can handle label_id out of defined labels case here.

Reply:
I agree with @BlairLee on this, I wouldn't remove this error check unless you're 100% sure that the data is always going to be there. This check should not be hurting anything, this isn't performant code to start with

Reply:
Yeah I think I removed it to test something, there is no reason not to have it